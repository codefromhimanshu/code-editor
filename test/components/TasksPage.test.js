// tests/TasksPage.test.tsx
import React from "react";
import renderer from "react-test-renderer";
import TasksPage from "../../src/components/TaskPage";

describe("TasksPage", () => {
  it("renders correctly", () => {
    const component = renderer.create(<TasksPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  // More tests could be added here
});
