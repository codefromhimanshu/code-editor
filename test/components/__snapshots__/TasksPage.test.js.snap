// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TasksPage renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="flex flex-row justify-between items-center mb-4"
  >
    <div
      className="sm:mb-0"
    >
      <select
        className="form-select block w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-primary-500 focus:border-primary-500"
        onChange={[Function]}
      >
        <option>
          All
        </option>
        <option>
          To Do
        </option>
        <option>
          In Progress
        </option>
        <option>
          Done
        </option>
      </select>
    </div>
    <button
      className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded inline-flex items-center"
      onClick={[Function]}
    >
      Create Task ✏️
    </button>
  </div>
  <div
    className="flex justify-between mb-4"
  >
    <input
      className="border p-2 rounded"
      onChange={[Function]}
      placeholder="Search tasks..."
      type="text"
      value=""
    />
    <select
      className="border p-2 rounded"
      onChange={[Function]}
      value="asc"
    >
      <option
        value="asc"
      >
        Sort Ascending
      </option>
      <option
        value="desc"
      >
        Sort Descending
      </option>
    </select>
  </div>
  <div
    className="space-y-4"
  >
    <div
      className="flex items-center""
    >
      <span>
         No Task present
      </span>
    </div>
  </div>
</div>
`;
